src/Card.tsx

```diff
@@ -8,9 +8,9 @@ export interface CardPropsBase<T> {
    */
   title?: string;
   /**
-   * Icon to the top-left of the title.
+   * Icon to the top-left of the title. Can be a ReactNode or a string equal to an image source.
    */
-  icon?: ReactNode;
+  icon?: ReactNode | string;
```

Comment:
[`ReactNode`](https://reactnative.dev/docs/react-node) already includes string:
```ts
type ReactText = string | number;
type ReactChild = ReactElement | ReactText;
...
type ReactNode = ReactChild | ReactFragment | ReactPortal | boolean | null | undefined;
```

Reply:
Good catch. I was trying to make it clearer to users they can pass a URL to an Icon but that's more about the docs than the types.